jetty-server/src/main/java/org/eclipse/jetty/server/HttpOutput.java

```diff
@@ -770,18 +770,11 @@ public void write(byte[] b, int off, int len) throws IOException
             _written = written;
 
             // Should we aggregate?
-            if (aggregate)
+            if (aggregate && !flush)
```

Comment:
This is a change of behaviour so that if we have aggregated bufferSize-1 bytes and then write 2 bytes we will flush the aggregate buffer and then aggregate the 2 bytes.   Previously we would aggregate 1 byte, flush aggregate buffer and then aggregate 1 byte.

I'm not necessarily against this change... and the code is simpler... but I'm cautious about more unintended consequences.... so perhaps we make that simplification in 10 and keep the logic the same in 9.